packages/nitro-protocol/gas-benchmarks/benchmark.test.ts

```diff
@@ -1,31 +1,31 @@
-import {channelId} from './fixtures';
+import {channelId, finalizationProof, finalState} from './fixtures';
 import {gasRequiredTo} from './gas';
 import {erc20AssetHolder, ethAssetHolder, nitroAdjudicator, token} from './vanillaSetup';
 
 describe('Consumes the expected gas for deployments', () => {
-  it(`when deploying the NitroAdjudicator >>>>>  ${gasRequiredTo.deployInfrastructureContracts.vanillaNitro.NitroAdjudicator} gas`, async () => {
```

Comment:
This change (removing the interpolated gas value) is in the spirit of maintaining a single "look here to understand the gas costs" point in `./gas.ts`?

Reply:
Partly, yes. It was also just getting annoying putting these here, especially when a test case covered multiple gas consumption values.